elmslie-compose/src/main/java/vivid/money/elmslie/compose/ElmComponentActvitiy.kt

```diff
@@ -0,0 +1,29 @@
+package vivid.money.elmslie.compose
```

Comment:
ElmComponent**Actvitiy**.kt typo should be ElmComponent**Activity**.kt

Reply:
Thank you, fixed